src/connectors/refinement-list/connectRefinementList.js

```diff
@@ -236,7 +242,12 @@ export default function connectRefinementList(renderFn, unmountFn) {
         };
 
         helper
-          .searchForFacetValues(attribute, query, limit, tags)
+          .searchForFacetValues(
+            attribute,
+            query,
+            isShowingMore ? showMoreLimit : limit,
```

Comment:
We can update and use the `getLimit` function.

Reply:
`this.getLimit()` is not available at this step.

Reply:
Yes but I don't think it's a reason to not use it. We could extract the function to re-use it.